lib/zen_to_i/constant.rb

```diff
@@ -2,9 +2,12 @@
 class String
   module Zen2Num
     KANNUM = { '〇' => 0, '' => 1,'一' => 1, '二' => 2,'三' => 3, '四' => 4,
-      '五' => 5, '六' => 6, '七' => 7, '八' => 8, '九' => 9 }
-    KANORDER = {'十' => 10,'百' => 100, '千' => 1000 , '' => 1}
-    KANORDER_F = { '万' => 10**4 , '億' => 10**8, '兆' => 10**12 , ''=> 1}
+      '五' => 5, '六' => 6, '七' => 7, '八' => 8, '九' => 9,
+      '零' => 0, '壱' => 1, '弐' => 2, '参' => 3, '肆' => 4,
+      '伍' => 5, '陸' => 6, '漆' => 7, '捌' => 8, '玖' => 9 }
+    KANORDER = {'十' => 10,'百' => 100, '千' => 1000 , '' => 1,
+      '拾' => 10, '陌' => 100, '佰' => 100, '阡' => 1000, '仟' => 1000 }
+    KANORDER_F = { '万' => 10**4 , '億' => 10**8, '兆' => 10**12 , ''=> 1, '萬' => 10**4 }
```

Comment:
__[rubocop]__
* Space found before comma.
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)